src/lib/common/networks/networks.ts

```diff
@@ -25,4 +26,27 @@ export const NetworkIdentifiers: {
   3: BridgeNetworks.Ethereum,
   4: BridgeNetworks.solana,
```

Comment:
We need to fix capitalization consistency.  Might as well start now.  Can you replace solana and algorand Bridge networks with Capitalized versions?

Reply:
@Elaixzar should i change `algorand` to `Algorand` and `solana` to `Solana`?